packages/components/src/image-select/ImageSelect.js

```diff
@@ -74,7 +91,7 @@ export default ImageSelect;
 ImageSelect.propTypes = {
 	defaultImageUrl: PropTypes.string,
 	imageUrl: PropTypes.string,
-	imageAltText: PropTypes.string,
+	imageAltText: PropTypes.string.isRequired,
```

Comment:
I think it is good to set the alt text to be required, since images should always have an alt text. 

However, we should either create an issue to add this alt text in the places where the `ImageSelect` component is used, or fix it in this PR as well.

Reply:
(Not sure if there is already an issue for it somewhere)

Reply:
Talked to Josee, there is an issue: https://yoast.atlassian.net/browse/IM-184